api/src/config.py

```diff
@@ -6,7 +6,10 @@
 
 class Config(BaseSettings):
     ENVIRONMENT: str = Field("local", env="ENVIRONMENT")
+
+    # Logging
     LOGGER_LEVEL: str = Field("INFO", env="LOGGING_LEVEL", to_lower=True)
+    APPINSIGHTS_CONSTRING: str = Field(None, env="APPINSIGHTS_CONSTRING")

```

Comment:
Considering it can be (and is defaulted to) `None`, the typing should show that. 
```suggestion
    APPINSIGHTS_CONSTRING: str | None = Field(None, env="APPINSIGHTS_CONSTRING")
```
or
```suggestion
    APPINSIGHTS_CONSTRING: Optional[str] = Field(None, env="APPINSIGHTS_CONSTRING")
```